src/index.ts

```diff
@@ -87,6 +87,10 @@ const authController = new AuthController(config.getAccountsMap())
 const app: Express = express()
 const port = config.getExpressPort()
 
+// to undify date format and strings
+// returns the current time as an ISO 8601 string
+const now = () => new Date().toISOString()
```

Comment:
@mre added this to make sure that local settings can't change the output format

Reply:
good idea, thanks